charts/dapr/charts/dapr_operator/values.yaml

```diff
@@ -35,6 +35,8 @@ ports:
 
 resources: {}
 
+extraEnvVars: []
```

Comment:
Make this a dictionary of key->value

Reply:
usually this is an array, at least in bitnami charts. I can see the benefit of using a dictionary, and thought about it, but that also includes adding a range to parse it into an array of dictionary in the template. I did not see env vars added in a chart in dapr that I looked for to follow so decided to follow bitnami's standard that is easier to implement.

I mentioned the format here https://github.com/dapr/dapr/issues/7182

Reply:
We have other fields in the chart that allow setting annotations and are dictionaries.

Using an array allows doing things like referencing secrets. However secrets can't be mounted in the current Helm chart so it doesn't matter in this case.

Using a dictionary is a lot simpler to pass via `--set` in Helm too.

My .02.

Reply:
annotations are dictionaries by default so it kind of make sense. true about `--set` but now helm also has `--set-json` that makes life easier for most of previous hard cases

Reply:
I could add the dictionary. let me think of a way to not make it to bad in the template

Reply:
@ItalyPaleAle could you do another check